runners/google-cloud-dataflow-java/src/test/java/org/apache/beam/runners/dataflow/util/CloudObjectsTest.java

```diff
@@ -112,7 +121,25 @@ public void defaultCodersIncludesCustomCoder() {
                       KvCoder.of(VarLongCoder.of(), ByteArrayCoder.of()),
                       IntervalWindow.getCoder()))
               .add(VarLongCoder.of())
-              .add(ByteArrayCoder.of());
+              .add(ByteArrayCoder.of())
```

Comment:
Done.

The `DefaultsPresentTest` should fail if there's any coder with a custom serializer/deserializer that isn't exercised by the `DefaultCoders` test.